modules/eks/eks.tf

```diff
@@ -10,6 +10,7 @@ locals {
       "volume_size", worker.volume_size,
       "kubelet_extra_args", <<EOT
 --node-labels sighup.io/cluster=${var.cluster_name},sighup.io/node_pool=${worker.name},%{for k, v in worker.labels}${k}=${v},%{endfor}
+%{if length(worker.taints) > 0}--register-with-taints %{for t in worker.taints}${t},%{endfor}%{endif}
```

Comment:
![Think](https://media.giphy.com/media/JTaJ0sO6sV0CAChkRq/giphy.gif)

Reply:
😂  i hope strongly @lnovara that you copied it from somewhere, otherwise, takes 1 week of  vacation more

Reply:
Kind-of, the work done by @angelbarrera92 helped a lot.